### DISCUSSION TITLE
Support Community Dark Mode?

### DISCUSSION THREAD

**Discussion by sudoneuron** [community/community#22192] [Accessibility]
<p>This is just a thought, but after spending some time in dark mode (which I have really liked — there’s always room for improvements but that’s a bunch of other discussions already taking place), I decided to jump over to this Support Community page to see what others had to say. I found it somewhat shocking to my eyes to go from dark mode to the bright white background of the Support Community, and I’m wondering if there are plans to add Dark Mode Beta to this side of GitHub, or what others might think about think about that.</p>

**Comment by gaknoll** [comment]
<p>Thanks for sharing the suggestion <a class="mention" href="/ahimsacode">@ahimsacode</a>! We have plans to expand dark mode support for additional products/pages/views as we move through our beta. The Community site uses a different platform than <a href="http://GitHub.com">GitHub.com</a> but I’ve shared this request with the Community team for their review!</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hey <a class="mention" href="/ahimsacode">@ahimsacode</a>,</p>
<p>Thank you for joining the community! We are happy you are here! You can change your community forum display preference to dark mode, by going to preferences&gt;theme&gt;dark :grinning_face_with_smiling_eyes:</p>
<p><img src="https://user-images.githubusercontent.com/20666190/181069422-9bd076a3-6d1a-4e6c-8b0b-d92b46fbfb63.jpeg" alt="EovqqapXIAESLdy" data-base62-sha1="tIzg3MlgGfiSTuURtGk8idiVjNB" width="680" height="255"></p>
<p>I hope this helps!<br>
AG</p>

**Comment by sudoneuron** [comment]
<p>Thanks <a class="mention" href="/AndreaGriffiths11">@AndreaGriffiths11</a>!</p>

**Comment by cehfisher** [comment]
Hi @sudoneuron - thank you for providing this feedback! After reviewing your issue report against the latest UI, this issue appears to be resolved now. Could you please confirm and close this issue? Thank you! (cc @queenofcorgis)
